Makefile

```diff
@@ -1,6 +1,6 @@
 GO ?= go
 GOLANGCI_LINT ?= $$($(GO) env GOPATH)/bin/golangci-lint
-GOLANGCI_LINT_VERSION ?= v1.47.2
+GOLANGCI_LINT_VERSION ?= v1.50.0
```

Comment:
update to [v1.50.1](https://github.com/golangci/golangci-lint/releases/tag/v1.50.1)